components/PageControl/examples/PageControlTypicalUseExample.swift

```diff
@@ -20,47 +20,51 @@ import MaterialComponents.MaterialPageControl
 
 class PageControlSwiftExampleViewController: UIViewController, UIScrollViewDelegate {
 
+  static let pageColors = [
+    MDCPalette.cyan.tint300,
+    MDCPalette.cyan.tint500,
+    MDCPalette.cyan.tint700,
+    MDCPalette.cyan.tint300,
+    MDCPalette.cyan.tint500,
+    MDCPalette.cyan.tint700
+  ]
+
   let pageControl = MDCPageControl()
   let scrollView = UIScrollView()
-  let pages = NSMutableArray()
+  let pages: [UIView] = PageControlSwiftExampleViewController.pageColors.enumerated().map {
```

Comment:
Since these are actually labels, let's make it `let pageLabels: [UILabel} = ...` and refer to it as a `pageLabel` in all local variables.

Reply:
Done